### PR TITLE
Issue/6

### PR DESCRIPTION
**Pull request by triggerdude33**
Did a change which hides cost-centeres which have empty names and have 0 income and expenses. Works well when i run it locally. I haven't tampered anything with the database in order for it to work. I'm a bit afraid that this change will break the system if it is pushed to main, since I don't fully understand how this system works. So please take a detailed look at it.

### CODE REVIEW

File: app/Models/CostCentre.php [hunk 1]
```diff
@@ -92,6 +92,13 @@ public function getInternalAttribute() {
 			return $x->balance;
 		});
 	}
+	
+	public function getDeletedAttribute() {
+		if(!strcmp($this->name, '') && $this->expenses === 0 && $this->income === 0)
```
Comment (Herkarl) [inline app/Models/CostCentre.php:97]: would be clearer with `strcmp($this->name, '') != 0`, since `strcmp` returns -1/0/1 rather than true/false

**Comment by Herkarl** [comment]
Haven't tested it locally, but it seems quite reasonable and consistent with what I remember from the implementations from the other PHP apps in our organisation. I find it hard to believe this would break anything on budget if it were to be pushed. 

Might be worth looking into if it would affect cashflow though.

I feel like `app.js` shouldn't be in the repo at all, since it seems like it is auto-generated.

File: app/Models/CostCentre.php (hunk 1 above)
Comment (triggerdude33) [reply app/Models/CostCentre.php:97]: looks good, can make a commit later

**Comment by triggerdude33** [comment]
Don't think cashflow will be affected. This will only change the visibility on budget. Empty cost centres will still be shown on cashflow

**Comment by Herkarl** [comment]
Sorry for the late second review. Tested this locally, and I couldn't get it to make any difference in what displayed (cloned the database from sips), might have done something wrong though.

@TerraDOOM made a PR (#12), that implemented a manual toggle for inactive budgets, thinking about if that could replace this one?
Once again, should have checked this out properly earlier, sorry for that.

**Comment by Herkarl** [comment]
For the record:
From my testing, this shouldn't break cashflow.
It does add an element in some JSON-objects returned from the API, but from what we could see, cashflow does not require the API responses to follow an exact format.

**Review by elmaxe — APPROVED** [review]

**Comment by DouglasFischer** [comment]
Fixed in #12

**Comment by DouglasFischer** [comment]
And in #15

**Pull request closed without merging**